tests/e2e/data/observatory/catalog/test_download.py

```diff
@@ -40,13 +40,16 @@ def setup_method(self):
             self.username = creds['USERNAME']
             self.base_url = creds['USERURL']
 
-        self.credentials = Credentials(username=self.username, api_key=self.apikey, base_url=self.base_url)
+        self.credentials = Credentials(username=self.username,
+                                       api_key=self.apikey,
+                                       base_url=self.base_url)
 
         self.tmp_file = file_path('tmp_file.csv')
 
     def teardown_method(self):
         if os.path.isfile(self.tmp_file):
-            os.remove(self.tmp_file)
+            print(self.tmp_file)
+            # os.remove(self.tmp_file)
```

Comment:
I believe this was used for development of tests but should be removed (should not be a big deal)

Reply:
dammit!